cumulusci.yml

```diff
@@ -262,4 +286,26 @@ plans:
                     case_behavior_record_types:
                         name: EDA - Case Behavior Record Types
                 options:
-                    unmanaged: false
+                    unmanaged: False
+            4:
+                task: deploy_einstein_templates
+                ui_options:
+                    is_required: False
+                checks:
+                    - when: "not tasks.has_einstein_licenses()"
+                      action: skip
+                      message: Skipped because Einstein Analytics permission sets are not available.
+
+
+    ra_einstein_template:
+        slug: ra_template
+        title: Install Education Cloud Recruiting Analytics Template
+        is_listed: False
+        tier: additional
+        steps:
+            1:
+                task: deploy_einstein_templates
+        checks:
+            - when: "not tasks.has_einstein_licenses()"
+              action: error
+              message: Einstein Analytics permission sets are not available.
```

Comment:
@jstvz if you want you can put the `checks` in the `deploy_einstein_templates` task config so you don't have to duplicate it.